### DISCUSSION TITLE
Transit ruler align to inner circle

### DISCUSSION THREAD

**Discussion by theodeluxe23** [Kibo/AstrologyChart2#2] [Ideas]
Hi Kibo,

First of all thanks for the really great AstroChart and AstroChart2! I have several ideas to share with you, but one of them is the request to change the transit rulers from the outside circle to the inner one. This would be more easier to find the degrees.

I'd love to chat with you about enhancements, since this would mean a great help for me.

Thanks
Theo

**Comment by Kibo** [comment]
Hi Theo
thank you a lot for your message. I think, it is a good idea.

How do you use the AstroChart? Do you have an static web app? Or do you have a ephemeris server for data (API)?

Tom

**Comment by theodeluxe23** [comment]
Hi Tom,

Thank you. I have some more questions and ideas in regards to AstroChart.
I would love to get in touch with you and talk about them.

Looking forward
Theo
